docs/static.md

```diff
@@ -98,5 +119,34 @@ script to customize the static build:
 * `CLEAN`: when set, libphp and all its dependencies are built from scratch (no cache)
 * `NO_COMPRESS`: don't compress the resulting binary using UPX
 * `DEBUG_SYMBOLS`: when set, debug-symbols will not be stripped and will be added within the binary
-* `MIMALLOC`: (experimental, Linux-only) replace musl's mallocng by [mimalloc](https://github.com/microsoft/mimalloc) for improved performance
+* `MIMALLOC`: (experimental, Linux-only) replace musl's mallocng by [mimalloc](https://github.com/microsoft/mimalloc) for improved performance. We only recommend using this for musl targeting builds, for glibc prefer disabling this option and using LD_PRELOAD when you run your binary instead.
 * `RELEASE`: (maintainers only) when set, the resulting binary will be uploaded on GitHub
+
+## Extensions
+
+With the glibc or mac based binaries, you can load PHP extensions dynamically. However, these extensions will have to be compiled with ZTS support.
+Since most package managers do not currently offer ZTS versions of their extensions, you will have to compile them yourself.
+
+For this, you can create a the static-builder-gnu Docker container, remote into it and compile the extensions with `./configure --with-php-config=/go/src/app/dist/static-php-cli/buildroot/bin/php-config`.
+
+Example steps:
```

Comment:
```suggestion
Example steps for [the Xdebug extension](https://xdebug.org):
```

Reply:
For what it's worth, we're (probably) going to integrate this into SPC. But until we're done with that, users can do it manually.
https://github.com/crazywhalecc/static-php-cli/pull/673#issuecomment-2747185241

Reply:
Nice!